cfme/tests/cli/test_appliance_console.py

```diff
@@ -25,6 +25,13 @@
     TZ('Pacific/Apia', ('10', '1')),
     TZ('UTC', ('11',))
 ]
+CLEAR = ''
```

Comment:
**REQUIRED:** Perhaps we should name this ```NEWLINE```, or ```RETURN```

Reply:
Agreed.